.github/workflows/pr.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v4
         with:
-          go-version: 1.19
+          go-version: '1.20'
```

Comment:
We should be using 1.20 for GA everywhere. 1.19 support is almost guaranteed when we CI/CD against 1.20